fluidBuild.config.cjs

```diff
@@ -202,6 +202,68 @@ module.exports = {
 		],
 	},
 
+	repoLayout: {
+		packages: [
+			"common/build/build-common",
+			"common/build/eslint-config-fluid",
+			"common/build/eslint-plugin-fluid",
+			"common/lib/common-utils",
+			"common/lib/protocol-definitions",
+			"tools/api-markdown-documenter",
+			"tools/benchmark",
+			"tools/changelog-generator-wrapper",
+			"tools/getkeys",
+			"tools/test-tools",
+		],
```

Comment:
Maybe we can get rid of these. While we migrate, the code could read the package setting from the old repo packages config, which would eventually be completely deprecated.

We would move independent packages into the main workspace as single-package release groups where needed. Eventually all the individual packages would be gone and we remove the old settings altogether.